rust/src/graph/mod.rs

```diff
@@ -201,18 +195,15 @@ impl GraphWrapper {
     }
 }
 
-/// Wrapper around the Graph struct that integrates with Python.
 #[pymethods]
-impl GraphWrapper {
+impl Graph {
```

Comment:
Not sure about this one. My thinking with having a wrapper class (maybe it should be called `PyGraph`) is so we can potentially free ourselves from the GIL if we need to, at some point, e.g. by passing around immutable references to the graph in pure Rust without having to think about Python.

Reply:
Fair! I'll remove this bit of the refactor.